app/src/main/res/layout/user_map_activity.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".UserMapActivity">
+    <fragment xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
```

Comment:
And if you've got the time it would be great to clean up a bit the redundant namespace as Cirrus said and the unused resources (even though now that I look at it more into details most of them don't seem to come from your code x)  )

Reply:
Ah, it was already said, mb haha